jetty9-base/src/main/jetty-config/jetty.commands

```diff
@@ -1,6 +1,6 @@
 --create-startd
 
---add-to-start=server,webapp,http,deploy,jsp,jstl,http-forwarded,resources,gae
```

Comment:
We can move those configurations to either modules/gcp.mod file as part of a fixed configuration, or we can generate them into a start.dgcp.ini file.

We loose a little niceness as by having them in jetty.commands they are used to substitute the param declarations in the appropriate ini files as they are generated.   So we will end up with the parameters in two ini files, however they will be commented out by default in the generated ini.

We can also make http-forwarded a dependency of the gcp module, so it is enabled transitively by gcp rather than explicitly here...  actually we could then use a gcp.commands file rather than directly generating the gcp.ini and the correct parameter substitution would take place.

I'll try some variations and see which is best... stand by...